hbase-common/src/main/java/org/apache/hadoop/hbase/HConstants.java

```diff
@@ -1040,10 +1040,12 @@ public enum OperationStatusCode {
   public static final String HFILE_PREAD_ALL_BYTES_ENABLED_KEY = "hfile.pread.all.bytes.enabled";
   public static final boolean HFILE_PREAD_ALL_BYTES_ENABLED_DEFAULT = false;
 
-  /*
-   * Minimum percentage of free heap necessary for a successful cluster startup.
+  /**
+   * Configuration key for the absolute amount of heap memory that must remain free for a
+   * RegionServer to start
    */
-  public static final float HBASE_CLUSTER_MINIMUM_MEMORY_THRESHOLD = 0.2f;
```

Comment:
Can we restore this constant? HConstants is `@InterfaceAudience.Public` so we should not remove the existing ones.